demo/cli/src/lib.rs

```diff
@@ -64,6 +64,14 @@ impl extrinsic_pool::api::ExtrinsicPool<UncheckedExtrinsic, BlockId, Hash> for D
 	{
 		Err("unimplemented".into())
 	}
+
+	fn light_status(&self) -> extrinsic_pool::txpool::LightStatus {
+		unreachable!()
```

Comment:
It is not supposed to be called in tests.

Reply:
yeah, that's true. Because of this I skipped rpc tests case (substrate/rpc/src/author/tests.rs). But isnt' this code is different in that it should be eventually implemented?

Reply:
`demo/cli` is about to be rewritten soon anyway.